bindings/java/src/main/native/Makefile.am

```diff
@@ -0,0 +1,48 @@
+JDK = $(JAVA_HOME)
+topdir=../../../../..
+javadir=$(topdir)/bindings/java
+MVN=@cd $(javadir) && mvn
+ucm_lib=ucm/.libs
+ucs_lib=ucs/.libs
+uct_lib=uct/.libs
+ucp_lib=ucp/.libs
+sources=bridge.cc request_util.cc
+
+# additional include paths necessary to compile the C library
+AM_CPPFLAGS = -fPIC -DPIC -Werror \
```

Comment:
-fPIC -DPIC -Werror are compiler flags, not preprocessor flags, should use libjucx_la_CXXFLAGS
(try avoiding use of AM_xxFLAGS)